package-overrides/github/julianshapiro/velocity@1.1.2.json

```diff
@@ -5,5 +5,12 @@
     "velocity.min.js",
     "velocity.ui.js",
     "velocity.ui.min.js"
-  ]
+  ],
+  "shim": {
+    "velocity.ui": {
+      "deps": [
+        "github:julianshapiro/velocity"
```

Comment:
It's probably best to just reference the main entry point explicitly here.

Note also this request as you've written it wouldn't work against jspm 0.16 beta.